test/cli/autocorrect-non-final-methods-final-module/autocorrect-non-final-methods-final-module.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+cwd="$(pwd)"
+infile="$cwd/test/cli/autocorrect-non-final-methods-final-module/autocorrect-non-final-methods-final-module.rb"
+
+tmp="$(mktemp -d)"
+
+cp "$infile" "$tmp"
+
+cd "$tmp" || exit 1
+# Suppress all output because the ordering of errors about non-final methods is
+# dependent on hash table iteration order.
```

Comment:
Eh...maybe?  I guess we could accumulate the methods for which we have errors, and then sort those by `Loc`?

Reply:
yeah i think that we've always tried to make the output of sorbet deterministic. seeing something where we can't capture the output in a best because it's order independent seems like a footgun